dash/development/update_components.py

```diff
@@ -31,7 +31,7 @@ def booststrap_components(components_source):
     )
 
     cmd = shlex.split(
-        "npx lerna exec --scope *@({})* -- npm i".format(source_glob),
```

Comment:
During the dev and release processes I think this one should still be `npm i`, but on CI it'd be nice to use `ci` - may need a flag to switch between them?

Reply:
Addressed in https://github.com/plotly/dash/pull/1939/commits/96e547f9c077c135b105b89cd44c570d8265ed65.